nilearn/input_data/tests/test_nifti_labels_masker.py

```diff
@@ -106,6 +106,24 @@ def test_nifti_labels_masker():
                                    get_affine(fmri11_img))
 
 
+def test_nifti_labels_masker_with_nans():
```

Comment:
You don't test infinities, which tells me that this PR is more about NaNs than infinities.